src/scripts.js

```diff
@@ -26,18 +26,48 @@ const userInfoBody = document.getElementById('userInfoBody');
 const greeting = document.getElementById('helloUser');
 const stepGoal = document.getElementById('stepGoal');
 
-const userRepo = new UserRepository(userData);
-const randomId = Math.floor(Math.random() * userRepo.usersData.length) + 1;
-const randomUser = userRepo.findUser(randomId);
+// const userRepo = new UserRepository(userData);
+// const randomId = Math.floor(Math.random() * userRepo.usersData.length) + 1;
+// const randomUser = userRepo.findUser(randomId);
 
-userInfoBody.innerHTML = `ID: ${randomUser.id}<br>
-  Name: ${randomUser.name}<br>
-  Address: ${randomUser.address}<br>
-  Email: ${randomUser.email}<br>
-  Stride Length: ${randomUser.strideLength}<br>
-  Daily Step Goal: ${randomUser.dailyStepGoal}<br>
-  `
+import { fetchData } from './apiCalls'
+import UserHydration from './hydrationRepository';
 
-greeting.innerText = `Welcome, ${userRepo.findFirstName(randomId)}!`
+const userID = 33
+let allUsers, allHydration
 
-stepGoal.innerText = `Your step goal: ${randomUser.dailyStepGoal} versus Average step goal: ${userRepo.calcAvgStepGoal()}`
\ No newline at end of file
+// => wrap the promise all in a function and have it be called on
+// load
+
+Promise.all([fetchData('users'), fetchData('hydration')])
+  .then(data => {
+    allUsers = data[0].users
+    console.log('USER', data[0])
+    console.log('wohoo', data[1])
+    allHydration = new UserHydration(data[1].hydrationData)
+    console.log('usersARRAY', allUsers)
+    console.log('hydrationARRAY', allHydration)
+  })
+  .then(() => {
+    renderHydration()
+    //renderUserInfo()
+    //functionToManipulateDOM()
+  })
```

Comment:
***FROM MY UNDERSTANDING*** the reason why the console logs outside of the promise all -ex <console.log(allUsers)>
was because the promise was happening after the console.. some scoping issue. 
By creating a second .then - now we can use that to assign methods that will manipulate/display the fetched data.

I have some commented out functions as an example... they are legitimate.